src/vs/workbench/contrib/chat/browser/actions/chatCodeblockActions.ts

```diff
@@ -357,6 +358,7 @@ export function registerChatCodeBlockActions() {
 					id: MenuId.ChatCodeBlock,
 					group: 'navigation',
 					isHiddenByDefault: true,
+					when: ContextKeyExpr.not('terminalChatFocus')
```

Comment:
they don't work for us because we don't use a `IChatWidget`

Reply:
that is because `context` is undefined here

![Screenshot 2024-03-08 at 8 00 20 AM](https://github.com/microsoft/vscode/assets/29464607/d169c8bc-1a6a-4260-94bb-7817f7c0bfe4)

since it returns here 

https://github.com/microsoft/vscode/blob/6c6bcf374da88ce4c43fb9165c5707b88ef16645/src/vs/workbench/contrib/chat/browser/actions/chatCodeblockActions.ts#L576-L580

we could add our terminal chat controller code here, but that seems wrong